lib/enumerable/all.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_all?(arg = nil)
+    if block_given?
+      my_each { |element| return false unless yield(element) }
```

Comment:
Your method should pass test cases such as:

```
[1, 2i, 3.14].all?(Numeric)                       #=> true
[nil, true, 99].all?                              #=> false
```

but it is not doing so at the moment, when no argument or block is given or when a class is passed as an argument (numeric, integer, etc...)

Reply:
I changed the way `#my_all`  address Classes:

![image](https://user-images.githubusercontent.com/36519478/68626366-3c8bcb00-04a9-11ea-9bb8-36a9445cb4be.png)

Now it return the same as your test cases